addons/event/views/event_views.xml

```diff
@@ -719,6 +738,8 @@
                     <field name="event_id"/>
                     <field name="partner_id"/>
                     <field name="company_id"/>
+                    <separator/>
+                    <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
```

Comment:
We generally prefix names by the related item, here it would be ``filter_inactive``